lib/facter/self_service.rb

```diff
@@ -186,6 +186,19 @@
     { S0022: validity }
   end
 
+  chunk(:S0029) do
+    next unless PuppetSelfService.replica? || PuppetSelfService.postgres? || PuppetSelfService.primary?
+    # check if concurrnet connections to Postgres approaching 90% defined
+    approaching_limit = false
+    maximum = PuppetSelfService.max_connections.to_i
+    current = PuppetSelfService.cur_connections.to_i
+    percent_used = current / maximum * 100
+    if percent_used >= 90
+      approaching_limit = true
+      { S0029: approaching_limit }
+    end
```

Comment:
```suggestion
      { S0029: percent_used >= 90 }
    end
```
This can be re written as above without the if statement with the same result